src/com/cdd/bao/util/Util.java

```diff
@@ -934,7 +934,7 @@ public static String streamToString(InputStream in) throws IOException
 	{
 		Writer wtr = new StringWriter();
 		char[] buff = new char[1024];
-		Reader rdr = new BufferedReader(new InputStreamReader(in,UTF8));
+		Reader rdr = new BufferedReader(new InputStreamReader(in, UTF8));
```

Comment:
Nothing to do with this change. The caller is responsible for closing the stream. You could state this in the comment.

Reply:
That's implied by default: if it did the opposite, that would require a very big caveat...